tests_e2e/README.md

```
@@ -0,0 +1,199 @@
+- `accessibility.spec.ts` - WCAG a11y tests (14 tests)
+- `visual.spec.ts` - Screenshot regression tests (16 tests)
+- `network.spec.ts` - API/network tests (15 tests)
+- `edge-cases.spec.ts` - Edge case tests (20 tests)
+
+## Backend Issue
+
+The TypeScript Playwright tests are failing with:
+```
+sqlalchemy.exc.MissingGreenlet: greenlet_spawn has not been called
+```
+
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)